documentation/staging/content/userguide/managing-domains/accessing-the-domain/port-forward.md

```diff
@@ -0,0 +1,273 @@
+---
+title: "Use Port Forwarding"
+date: 2019-02-23T17:39:15-05:00
+draft: false
+weight: 4
+description: "You can use Port Forwarding to access the Administration console and WLST."
+---
+
+### Contents
+
+* [Overview](#overview)
+* [Set up Administration Server network channels for port forward access](#set-up-administration-server-network-channels-for-port-forward-access)
+* [Port forward to an Administration Server pod](#port-forward-to-an-administration-server-pod) 
+  * [Port forward example](#port-forward-example)
+  * [Port forward notes and warnings](#port-forward-notes-and-warnings)
+  * [Enabling WLST access when local and remote ports do not match](#enabling-wlst-access-when-local-and-remote-ports-do-not-match)
+  * [Terminating port forwarding](#terminating-port-forwarding)
+
+### Overview
+
+Beginning with WebLogic Kubernetes Operator version 3.3.2,
```

Comment:
This looks fine to me, but we should ask Rosemary to help us wordsmith this. There probably is some simpler way to express this. :)

Reply:
@rosemarymarano Once the ball is in your court, please see ^^^.

Reply:
@tbarnes-us @rjeberhard Sure. "...help us wordsmith this. There probably is some simpler way to express this.
One question: to what are you referring as "this," the Overview section or something else?

Reply:
I thought that the wording was fine.